windows.Dockerfile

```diff
@@ -0,0 +1,99 @@
+# escape=`
```

Comment:
Since the default escape character in Dockefile syntax is `\` which **is also** the directory path separator in **Windows** systems, we need to define the backtick as the default escape character (i.e. to escape newlines) as per[1].

[1] https://docs.docker.com/engine/reference/builder/#escape